crates/jws/src/lib.rs

```diff
@@ -479,7 +479,8 @@ mod tests {
             &header,
             &payload,
         )
-        .unwrap() + "123";
+        .unwrap()
```

Comment:
interesting, added https://github.com/TBD54566975/web5-rs/issues/200 to try and avoid this in the future